README.md

```diff
@@ -31,7 +31,7 @@ Help with available usage can also be found after installation with `makedeb --h
 See [Obtaining Support](https://docs.makedeb.org/support/obtaining-support) in the makedeb Docs.
 
 ## Donate
-[![Stand With Ukraine](https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/banner-personal-page.svg)](https://stand-with-ukraine.pp.ua)
+[![Stand With Ukraine](The.svg)](https://stand-with-ukraine.pp.ua)
```

Comment:
I understand wanting to support this whole situation with Ukraine, but I'd really like to keep this out of here for now. This isn't anything with me being against the mission or anything, I'm just trying to avoid any political drama with makedeb.